packages/graphql/server/resolvers/mutators.ts

```diff
@@ -43,24 +47,15 @@ import { throwError } from "./errors";
 import { ModelMutationPermissionsOptions } from "@vulcanjs/model";
 import { isMemberOf } from "@vulcanjs/permissions";
 import { getModelConnector } from "./context";
-import pickBy from "lodash/pickBy";
+import { FilterableInput } from "../../typings";
+import { deprecate } from "@vulcanjs/utils/dist";
```

Comment:
Remove the "/dist", sometimes it happens with the auto-import => that's the built version of the file and will create issues in production